docs/articles/documentation/recipes/best-practices/create-faster-tests.md

```diff
@@ -0,0 +1,101 @@
+
+To run tests in a headless browser, use the `:headless` CLI parameter:
+
+```sh
+testcafe "chrome:headless" tests/
+```
+
+For more information, see [Test in Headless Mode](../../guides/concepts/browsers.md#test-in-headless-mode)
+
+## Use Roles for Login
+
+If your tests require activity from logged in users, use [Roles](../../guides/advanced-guides/authentication.md#user-roles) for authentication.
+
+Put Roles in the [beforeEach hook](../../reference/test-api/fixture/beforeeach.md) of your test suite: following tests in a suite reuse authentication data and authentication happens instantly.
+
+With `Roles`, TestCafe remembers cookies associated with every logged-in account. When you switch Roles, TestCafe erases authentication data and you don't have to log out manually.
+
+If your test covers logins on multiple websites, the active Role collects the authentication data from all these websites. When you switch to that Role later, you are logged in to all these websites.
+
+Use an [Anonymous Role](../../guides/advanced-guides/authentication.md#anonymous-role) to instantly log out of all accounts during a test.
+
+> Roles can access authentication data in cookie and browser storage. If your authentication system stores data elsewhere, you may not be able to use roles.
+
+For more info on Roles, see [User Roles](../../guides/advanced-guides/authentication.md#user-roles)
+
+## Run Tests Concurrently
+
+Tests execute faster if run concurrently. In concurrent mode, TestCafe creates multiple instances of specified browsers and uses that browser pool to run tests.
+
+Use the [--concurency](../../reference/command-line-interface.md#-c-n---concurrency-n) CLI option to launch tests concurrently:
+
+```sh
+testcafe --concurency 3 chrome tests/
+```
+
+For more info on concurrency, read [Run Tests Concurrently](../../guides/basic-guides/run-tests.md#run-tests-concurrently).
+
+## Mock Requests
+
+A tested application may interact with remote resources (for example, an analytics service or a database). Requests to such resources may create delays in test suite execution. To avoid such delays, you can mock requests to these resources.
+
+TestCafe [request mocker](../../reference/test-api/requestmock/README.md) intercepts requests from your app to external resources and responds with data that you specify. A mocked request is resolved almost instantly, which eliminates a possible delay caused by a foreign resource or network lag.
+
+For more information about mocking requests, visit [Mock HTTP Requests](../../guides/advanced-guides/intercept-http-requests.md#mock-http-requests).
+
+## Optimize Your Page Model
+
+Page model is a great way to structure your test suite. However, wrong page model structure can increase testing time in large test suites.
```

Comment:
A test suite is a set of testing tools. Why can't we use the word 'fixture' ?
*Wrong* implies something that does not reflect reality. 'Improper' fits this particular sentence much better.

```suggestion
Page models offer a great way to outline your application. However, improper page model structure can increase testing time in large fixtures.
```

Reply:
[Test suite is a collection of tests.](https://softwaretestingfundamentals.com/test-suite/#post-2042:~:text=A%20TEST%20SUITE%20is%20a%20collection%20of%20test%20cases.)